### PR TITLE
Add CSP Headers

### LINKED ISSUE #2728: Implement AMP CSP headers
We should be a good example and serve our site with the correct CSP headers. 

See https://github.com/ampproject/amp.dev/pull/2726

### PR DESCRIPTION
**Pull request by bezoerb**
fixes #2728

### CODE REVIEW

**Review by sebastianbenz — COMMENTED** [review]
nice!

**Comment by matthiasrohmer** [comment]
As discussed offline: we need to disable nonces as rendered templates are cached.

File: platform/lib/middleware/csp.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (bezoerb) [reply platform/lib/middleware/csp.js:2]: :)

**Review by sebastianbenz — COMMENTED** [review]
One other thing: we need to configure the host based on the current environment (staging vs prod).

File: platform/lib/middleware/csp.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    workerSrc: [`'self'`, 'blob:'],
+    scriptSrc: [
+      'blob:',
+      `'unsafe-inline'`,
+      'https://cdn.ampproject.org/v0.js',
+      'https://cdn.ampproject.org/v0/',
+      'https://cdn.ampproject.org/sw/',
+      'https://cdn.ampproject.org/viewer/',
+      'https://cdn.ampproject.org/rtv/',
+      'https://www.googletagmanager.com/gtag/js',
```
[the remark this answers is not in the document]
Comment (bezoerb) [reply platform/lib/middleware/csp.js:34]: done

**Review by sebastianbenz — APPROVED** [review]
Thanks! Super happy to have this

**Pull request merged**